server/controllers/auth.js

```diff
@@ -26,31 +26,31 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
   const user = await User.create({
     name,
     email,
-    password
+    password,
```

Comment:
Should be no back-end changes in this ticket since it's front-end. Please revert this file with git. Thanks!